TramsDataApi.Test/Integration/AcademyTransferProjectIntegrationTests.cs

```diff
@@ -362,7 +362,8 @@ public async Task CanUpdateTheTransferringAcademiesOfAnAcademyTransferProject()
                     (List<TransferringAcademiesRequest>) Builder<TransferringAcademiesRequest>
                         .CreateListOfSize(2).All()
                         .With(ta => ta.IncomingTrustUkprn = randomGenerator.NextString(8, 8))
-                        .With(ta => ta.OutgoingAcademyUkprn = randomGenerator.NextString(8, 8)).Build())
+                        .With(ta => ta.OutgoingAcademyUkprn = randomGenerator.NextString(8, 8))
```

Comment:
There are a few tests in this file that don't include the new fields, not sure if they are 100% necessary but I'd just double check

Reply:
Updated all unit tests with the fields